state-chain/node/src/chain_spec/perseverance.rs

```diff
@@ -58,49 +58,24 @@ pub const SNOW_WHITE_SR25519: [u8; 32] =
 	hex_literal::hex!["84f131a66e88e3e5f8dce20d413cab3fbb13769a14a4c7b640b7222863ef353d"];
 
 pub fn extra_accounts() -> Vec<(AccountId, AccountRole, FlipBalance, Option<Vec<u8>>)> {
-	[
-		vec![
-			(
-				parse_account("cFMTNSQQVfBo2HqtekvhLPfZY764kuJDVFG1EvnnDGYxc3LRW"),
-				AccountRole::Broker,
-				1_000 * FLIPPERINOS_PER_FLIP,
-				Some(b"Chainflip Genesis Broker".to_vec()),
-			),
-			(
-				parse_account("cFN2sr3eDPoyp3G4CAg3EBRMo2VMoYJ7x3rBn51tmXsguYzMX"),
-				AccountRole::LiquidityProvider,
-				1_000 * FLIPPERINOS_PER_FLIP,
-				Some(b"Chainflip Genesis Liquidity Provider".to_vec()),
-			),
-		],
-		phoenix_accounts(),
-	]
+	[vec![
+		(
+			parse_account("cFMTNSQQVfBo2HqtekvhLPfZY764kuJDVFG1EvnnDGYxc3LRW"),
+			AccountRole::Broker,
+			1_000 * FLIPPERINOS_PER_FLIP,
+			Some(b"Chainflip Genesis Broker".to_vec()),
+		),
+		(
+			parse_account("cFN2sr3eDPoyp3G4CAg3EBRMo2VMoYJ7x3rBn51tmXsguYzMX"),
+			AccountRole::LiquidityProvider,
+			1_000 * FLIPPERINOS_PER_FLIP,
+			Some(b"Chainflip Genesis Liquidity Provider".to_vec()),
+		),
+	]]
 	.into_iter()
 	.flatten()
 	.collect()
 }
 
-#[ignore = "Only used as a convenience."]
-#[test]
-fn print_total() {
-	let s = phoenix_accounts().iter().map(|(_, _, s, _)| *s).sum::<u128>();
-	println!("{s} / {}", s / FLIPPERINOS_PER_FLIP);
-}
-
-// Remember to add some extra funds to the dwarves to ensure they remain in the authority set.
-fn phoenix_accounts() -> Vec<(AccountId, AccountRole, FlipBalance, Option<Vec<u8>>)> {
```

Comment:
Do we want to remove this?

Reply:
We don't need to keep it unless we want to keep the old balances and vanity names. (Which we might want to do - it's really not that hard). 

If we do delete it, we might as well delete the perserverance.snapshot file too.

Reply:
We aren't going to keep the accounts this time. Fresh network, fresh nodes, no airdrop /snapshot

Reply:
I would say let's keep it fresh. Have everything start a new